amaranth/vendor/xilinx.py

```diff
@@ -102,6 +102,20 @@ class XilinxPlatform(TemplatedPlatform):
 
     Available overrides:
         * ``add_constraints``: inserts commands in XDC file.
+
+    .. rubric:: Xray toolchain
+
+    Required tools:
+        * ``yosys``
+        * ``nextpnr-xilinx``
+        * ``fasm2frames``
+        * ``xc7frames2bit``
+
+    The environment is populated by running the script specified in the environment variable
+    ``AMARANTH_ENV_Xray``, if present.
+
+    Available overrides:
+        * ``nextpnr_dir``: overrides toolchain db path.
```

Comment:
I don't think we need an override for this; overrides are for compile-time knobs, not runtime ones.

Reply:
Done